luacode/formatter.lua

```diff
@@ -464,6 +459,13 @@ local delimiterposition = function (str, strstart)
     end
 end
 
+local function validate(string)
+  local fileName = os.tmpname()
+  writeFile(fileName, string)
+  local exitCode = os.execute("./lua53/lua53.sh ./luacode/validate.lua " .. fileName)
```

Comment:
Can you make this find the Windows Lua as well? It may be used by systems folks on parametric data and will be published publicly so we want to have it not fall down on Windows if it's not unreasonably difficult.